style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
+
+This page outlines the style guide for SCSS for the jQuery Chassis project.
+
+## Comments
+
+SCSS comments should follow the valid CSS comment guidelines outlined in the jQuery contribute CSS style guide.
+
+```scss
+/* Good single line comment */
+
+/*
+* Good example of a multi-line comment.
+* If your comment takes up multiple lines, please do this.
+*/
+```
+
+## Cascading
+
+Avoid cascading as often as possible and follow the `BEM` naming conventions. If cascading is necessary avoid cascading selectors that are more than 3 selectors long.
+
+```scss
+/* 
+* Example without BEM using cascading 
+* (Please avoid this)
```

Comment:
If we are going to use bem this should be more then a kind suggestion there should never be any cascading.